src/store/storage-controller.ts

```diff
@@ -111,19 +109,43 @@ export class StorageController {
     return messages;
   }
 
-  public static async delete(
-    messageStore: MessageStore,
-    dataStore: DataStore,
+  public async delete(
     tenant: string,
     message: BaseMessage
   ): Promise<void> {
     const messageCid = await Message.getCid(message);
 
     if (message.descriptor.dataCid !== undefined) {
-      await dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
+      await this.dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
     }
 
-    await messageStore.delete(tenant, messageCid);
+    await this.messageStore.delete(tenant, messageCid);
+  }
+
+  public get MessageStore(): MessageStore { return this.messageStore; }
+
+  public queryMessages(tenant: string, filter: Filter, options?: MessageStoreOptions): Promise<BaseMessage[]> {
+    return this.messageStore.query(tenant, filter, options);
+  }
+
+  public putMessage(tenant: string, messageJson: BaseMessage, indexes, options?): Promise<void> {
```

Comment:
maybe `putMessageWithoutData` so that it's super clear "we are adding to the `MessageStore` without adding to the `DataStore`, which probably should only be done in super specific circumstances"?

also can we move these to be more nearby the similarly named functions (e.g. `putMessageWithoutData` would go right above/below `putMessageWithData`, `queryMessages` would go right above/below `queryRecordsWrites`, etc.)?

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/42f36fb0385704079ff7f0a11a32f953d80a25c7